### DISCUSSION TITLE
Code breaks with 0.5

### DISCUSSION THREAD

**Discussion by troelsarvin** [tokio-rs/axum#912] [Q&A] [answered]
I have some test code which compiled and worked well with Axum 0.4, but I get compile errors when trying with Axum 0.5+
I don't understand the compile errors.
My code and the compile errors may be inspected here: https://troels.arvin.dk/rust/axum/1/
I've tried adding use axum_macros::debug_handler stuff to the code, because I saw in some posts that it might help making compile errors more digestable, but it hasn't helped in my case.

Can someone help me understand what I'm doing wrong?

**Comment by davidpdrsn** [comment]
I don't believe `askama_axum` has been updated. Try using it via a git dependency.

**Reply by troelsarvin** [reply]
I tried using git dependencies, but it didn't help. I've submitted https://github.com/djc/askama/issues/661

**Reply by jplatte** [reply]
Works for me, maybe this helps? https://github.com/jplatte/turbo.fish/commit/f0ea82797c73748d9ec59febd1be42c52693c5dc

**Comment by genusistimelord** [comment, marked as answer]
This issue was fixed within the main branch of Askama via the merged PR https://github.com/djc/askama/pull/662. Once DJC is ready to release the next version of Askama it will then be available. till then Please use the main branch in the git till it has been resolved within the crates.

**Comment by benwis** [comment]
I have no idea when the next release is planned, but is it possible to do an interim release. I spent far too long trying to make Askama work with Axum today, only to find out it's fundamentally broken

**Reply by genusistimelord** [reply]
I have a copy that i keep updated for Axum as i use it with the latest versions. Though it does change a lot when i need to test a debug batch for David. https://github.com/genusistimelord/askama

Honestly though Askama is not Axum and its not Axums problem to update and maintain Askama. Askama's maker has decided to not add any new Axum changes to the Repo till Axum is out in Stable.

**Reply by benwis** [reply]
@genusistimelord Quite right, I am not expecting Axum to do this, I was thinking Askama should release an interim version or at least note somewhere its not working in Stable

**Reply by benwis** [reply]
Thanks for the link and I apologize I thought this was the askama repo.
